index.html

```diff
@@ -20,7 +20,7 @@ <h1 class="page-title">Contrast Ratio Calculator</h1>
               </div>
               <div class="input-items">
                 <div class="swatch-one"></div>
-                <input id="foreground-color" for="foreground-color" name="foreground-color" type="text" value="#000000" />
+                <input id="foreground-color" for="foreground-color" name="foreground-color" type="text" />
```

Comment:
I see a `for` attribute here, is this a typo?, I think this attribute is reserved for `output` and `label` element.

Reply:
I am curious about this myself. I did not changed this part. That's from original HTML, written by @Kernix13

Reply:
You are reviewing closed PR by the way. Maybe you misunderstood my discord message? There is opened PR with small readme file changes, I was referring to that

Reply:
However, if you have thoughts about this PR, I am definitely interested to hear it